app/views/posts/index.html.erb

```diff
@@ -1 +1,27 @@
-<h1> here are all posts </h1>
\ No newline at end of file
+<div id="container_index_posts">
+  <div id="container_container">
+      <img id="container_container_img" src="<%= @user.Photo %>" alt="user profile">
+      <div id="container_container_div">
+        <h3><%= @user.Name %></h3>
+        <p>Number of posts: <%= @user.PostsCounter %></p>
+      </div>
+    </div>
+    <% @posts.each do |post|%>
```

Comment:
- [ ] Kindly render only 5 recent comments for each post as requested. The method `recent_commemnt` you implemented in the post model will do that perfectly 👍